test/helpers.py

```diff
@@ -195,9 +195,12 @@ def waitport(port, prog, perTickCheck=None):
                 raise
     raise(Exception("timed out waiting for debug port %d (%s)" % (port, prog)))
 
-def waithealth(prog, port, host_override):
+def waithealth(prog, port, host_override, fakeclock):
+    e = os.environ.copy()
+    if fakeclock:
+        e.setdefault("FAKECLOCK", fakeclock)
```

Comment:
Out of curiosity, with skew checking turned off for integration tests, why was this diff needed?

Reply:
Just because I didn't think to remove it! I'll double-check that it works without it and then revert this portion of the diff.